cypress/support/pages/cart.pageObject.js

```diff
@@ -0,0 +1,36 @@
+import PageObject from '../PageObject';
+
+class CartPageObject extends PageObject {
+  assertProductInCart(prodName) {
+    cy.get('tbody tr td:nth-child(2)')
+      .should('have.text', prodName);
+  }
+
+  placeOrder() {
+    cy.contains('button', 'Place Order')
+      .click();
+  }
+
+  fillTheForm(name, country, city, cardNo, month, year) {
+    cy.get('input#name').type(name);
+    cy.get('input#country').type(country);
+    cy.get('input#city').type(city);
+    cy.get('input#card').type(cardNo);
+    cy.get('input#month').type(month);
+    cy.get('input#year').type(year);
+    cy.contains('button', 'Purchase')
+      .click();
+  }
+
+  assertPurchaseInfo(name) {
+    cy.get('.sweet-alert p.text-muted')
+      .should('contain.text', 'Name: ' + name);
```

Comment:
Ensure that the selector `.sweet-alert p.text-muted` correctly targets the element containing the purchase information. If the structure of the alert changes, this selector might need to be updated.